USAGE.md

```diff
@@ -0,0 +1,80 @@
+# USAGE NOTES with Ubuntu 20.04 LTS
+
+## Configuration file
+
+*ScanTool* saves the settings of the configuration panel in a file named *.scantoolrc* located in the home directory. Example of configuration file:
+
+```ini
+version = 2.1
+
+[general]
+system_of_measurements = 0
+display_mode = 1
+
+[comm]
+baud_rate = 15
+comport_number = 4
+```
+
+To reset the configuration (e.g., when the program is stuck in resetting an unavailable hardware interface), simply remove the file:
+
+```bash
+rm ~/.scantoolrc
+```
+
+## Standard serial COM port
+
+A standard serial COM port in the form `/dev/ttyS<number>` is configured in *~/.scantoolrc* with the following line: `comport_number = <number>`; for instance, `/dev/ttyS3` is represented as follows in *~/.scantoolrc*:
+
+```ini
+[comm]
+comport_number = 3
+```
+
+## USB COM port
+
+A USB COM port in the form `/dev/ttyUSB<number>` is configured in *~/.scantoolrc* with the following line: `comport_number = ` *100 + USB number*; for instance, for */dev/pts/3*, set the following in *~/.scantoolrc*:
+
+```ini
+[comm]
+comport_number = 1003
+```
+
+## Pseudo-tty support
+
+To use a pseudo-tty as COM port, edit *~/.scantoolrc* and add the line `comport_number = ` *1000 + pty number*; for instance, for */dev/pts/3*, set the following in *~/.scantoolrc*:
+
+```ini
+[comm]
+comport_number = 1003
+```
+
+Note: when setting a pseudo-tty, only use *~/.scantoolrc* and do not edit the serial port through the *"Options"* panel of the user interface.
+
+## Compilation
```

Comment:
This and compilation notes will need updating after the requested changes to those options. Otherwise, yeah, this looks nice! Thanks! :)